google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -159,6 +165,13 @@ public Builder setFormatOptions(FormatOptions formatOptions) {
 
 
     @Override
+    public Builder setNullMarker(String nullMarker) {
+      this.nullMarker = nullMarker;
+      return this;
+    }
+
+
```

Comment:
nit: It does not seem consistent with other files to have two lines between methods.